src/UI/Tooltip.js

```diff
@@ -30,10 +29,9 @@ const checkPosition = function () {
     }
 
     this.animationFrame = window.requestAnimationFrame(() => {
-        const isEmailInput = getInputMainType(this.input) === 'emailNew'
-        // Placement for the email autofill tooltip is relative to the position of the Dax icon
-        const position = isEmailInput ? getDaxBoundingBox(this.input)
-            : this.input.getBoundingClientRect()
+        // In extensions, the tooltip is centered on the Dax icon
+        const position = isApp ? this.input.getBoundingClientRect()
+            : getDaxBoundingBox(this.input)
```

Comment:
Now for the browser app we always use the new tooltip with positioning based on the input rather than the icon.